src/utils/configParser.ts

```diff
@@ -27,7 +29,7 @@ export const getNetworkList = (config: Record<string, string | undefined>): stri
 }
 
 export const getDefaultChain = (config: Record<string, string | undefined>) => {
-  return getNetworkList(config)[0]
+  return localStorage.getItem(defaultChainKey) || getNetworkList(config)[0]
```

Comment:
we need to make sure that the one save in localstorage exist in env, otherwise - in case we abandon some network - we might get stuck here with a tiny bug